src/Http/Response.php

```diff
@@ -2233,6 +2233,21 @@ public function withFile($path, array $options = [])
     }
 
     /**
+     * Convenience method to set a string into the response body
+     *
+     * @param string $string The string to be sent
+     * @return static
+     */
+    public function withStringBody($string)
+    {
+        $new = clone $this;
+        $new->_createStream();
```

Comment:
It is not strange, that is the design of that class visibility. As a fun fact, protected in java means that any class in the same package (namespace) can access that property or method